apidocs/openapi/certs.yaml

```diff
@@ -81,16 +81,18 @@ paths:
           description: Database can't process request.
         "500":
           $ref: "#/components/responses/ServiceError"
+
+  /{domainID}/certs/{clientID}:
```

Comment:
Just a question here,
I am able to create multiple certs associated with a ClientID, during revocation should I not be able to revoke a single cert as well?